python/qpid_dispatch_internal/management/agent.py

```diff
@@ -417,6 +417,13 @@ def create(self):
     def __str__(self):
         return super(LinkRouteEntity, self).__str__().replace("Entity(", "LinkRouteEntity(")
 
+class RouterConnectionAttachsubscriptionEntity(EntityAdapter):
```

Comment:
Why is this even mentioned in the Python management code?  It can't be configured and management commands are handled in the router core.

Reply:
Error reporting.  Without it the router fails with a generic parse error. With this we get:

qdrouterd: Python: CError: Configuration: attachSubscription cannot be created via the configuration file

I'm going to have to re-work this on changing the name in any case.  I can pull it if you want.

Reply:
Ya know, the default error message isn't too bad:

2018-10-19 15:37:07.199514 -0400 ERROR (error) Python: Exception: Cannot load configuration file X.conf: No such entity type 'org.apache.qpid.dispatch.attachSubscription'
2018-10-19 15:37:07.199789 -0400 ERROR (error) Traceback (most recent call last):
  File "/home/kgiusti/work/dispatch/qpid-dispatch/python/qpid_dispatch_internal/management/config.py", line 169, in configure_dispatch
    config = Config(filename)
  File "/home/kgiusti/work/dispatch/qpid-dispatch/python/qpid_dispatch_internal/management/config.py", line 56, in __init__
    % (filename, e))
Exception: Cannot load configuration file X.conf: No such entity type 'org.apache.qpid.dispatch.attachSubscription'

2018-10-19 15:37:07.199805 -0400 MAIN (critical) Router start-up failed: Python: Exception: Cannot load configuration file X.conf: No such entity type 'org.apache.qpid.dispatch.attachSubscription'
qdrouterd: Python: Exception: Cannot load configuration file X.conf: No such entity type 'org.apache.qpid.dispatch.attachSubscription'

A little better than a generic parse error.

Reply:
I think we can dispense with all this work just to improve the error message.  We should make sure that the man page for the configuration file doesn't mention this type.